RocketChat/Rocket.Chat.ReactNative
# [FIX] Google OAuth triggering cookies logic

## Proposed changes

Adds an attribute that help us differentiate when the oauth is coming from web view or not, that way we can log out without the need of clearing cache when we logged in with Google OAuth.

## Issue(s)	


## How to test or reproduce


Test case 1

1. Log in with Google OAuth
2. Go to Settings
3. Tap on `Logout`
4. Tap on `Logout` from the modal

It should `Logout` without raising the alert for clearing cookies

Test case 2

1. Log in with SAML
2. Go to Settings
3. Tap on `Logout`
4. Tap on `Logout` from the modal

It should raise the alert for clearing cookies

## Screenshots

## Types of changes




## Checklist



## Further comments
